content_structure/test_guide.md

```diff
@@ -488,3 +489,39 @@ When it is shared it can be displayed in another tenant, but not edited.
 * Test that the playlist is displayed on the screen. See T1 for how to activate the screen.
 
 ---
+
+## T9 Feed sources
+
+### Description
+
+Feed sources are the link between external data source and slides. An example is the RSS feed.
+When creating a slide with the RSS template a feed source ("datakilde") needs to be selected.
+After the feed source has been selected it is possible to input the url to the rss feed to show data from.
+The feed source contains the code that calls the external source and converts data to a format the RSS template
+understands.
+
+Feed sources are connected to a tenant. New feed sources can be connected to a tenant in through the
+"Datakilde" link in the navigation.
+
```

Comment:
If you want a new feed source on the list you need to contact a developer

Reply:
I do not think this is relevant for the guide.